src/main/java/simpli/tasks/Event.java

```diff
@@ -0,0 +1,39 @@
+package simpli.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+    protected LocalDateTime from;
```

Comment:
I noticed that the from and to fields in the Event class are protected. Would making them private and using getters enhance encapsulation?